features/service-operation.feature

```diff
@@ -0,0 +1,36 @@
+Feature: Command output from service operations like stop/start/restart
+  service-manager should return the correct exit code from stop/start/restart command
+
+  @operation
+  Scenario Outline: Boot and execute service operations like stop/start/restart
+    Given box is <box>
+    And provider is <provider>
+    And a file named "Vagrantfile" with:
+    """
+    require 'vagrant-libvirt'
+
+    Vagrant.configure('2') do |config|
+      config.vm.box = '<box>'
+      config.vm.box_url = 'file://../boxes/<box>-<provider>.box'
+      config.vm.network :private_network, ip: '<ip>'
+      config.vm.synced_folder '.', '/vagrant', disabled: true
+      config.servicemanager.services = 'docker'
+    end
+    """
+
+    When I successfully run `bundle exec vagrant up --provider <provider>`
+      And I successfully run `bundle exec vagrant service-manager stop docker`
```

Comment:
systemctl with docker:

    systemctl status docker
    ● docker.service - Docker Application Container Engine
       Loaded: loaded (/usr/lib/systemd/system/docker.service; enabled; vendor     preset: disabled)
      Drop-In: /usr/lib/systemd/system/docker.service.d
               └─flannel.conf
       Active: active (running) since Fri 2016-05-20 06:29:18 EDT; 2 days ago
         Docs: http://docs.docker.com
     Main PID: 12949 (sh)
       Memory: 115.0M
       CGroup: /system.slice/docker.service
               ├─12949 /bin/sh -c /usr/bin/docker daemon $OPTIONS                $DOCKER_STORAGE_OPTIONS            $DOCKER_NETWORK_OPTIONS                $ADD_REGISTRY            $BLOCK_REGISTRY                $INSECURE_REGISTRY            2>&1 | /usr/bin/forward-journald -tag     dock...
               ├─12951 /usr/bin/docker daemon --selinux-enabled -H tcp://0.0.0.0:2376     -H unix:///var/run/docker.sock --tlscacert=/etc/docker/ca.pem     --tlscert=/etc/docker/server-cert.pem --tlskey=/etc/docker/    server-key.pem --tlsverify --storage-driver devicemapper --stora...
               └─12952 /usr/bin/forward-journald -tag docker

You are getting among other things the main pid. For the restart you can confirm that the pid changes.